packages/react-scripts/config/webpack.development.js

```diff
@@ -0,0 +1,62 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const path = require('path');
+const paths = require('./paths');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
+const WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
+
+// We are in the development config. This contains development only features.
+module.exports = function() {
+  return {
+    devtool: 'cheap-module-source-map',

```

Comment:
`eval-cheap-module-source-map` would be faster

Reply:
We had an issue when we switched to eval-source-map due to mismatch line numbers in react-error-overlay.  I don't remember whether I tried eval-cheap-module-source-map.
See https://github.com/webpack/webpack/issues/8910